app/javascript/theme/components/modals/_login.scss

```diff
@@ -3,6 +3,12 @@
     margin-bottom: 0.75rem;
   }
 
+  &__support {
```

Comment:
Let's move this to `theme/shared/_form-input.scss` as `&__instructional-copy` or a similar generic class name.

Reply:
Should also be nested under `FormInput`.